src/wp-includes/general-template.php

```diff
@@ -2070,7 +2070,15 @@ function wp_get_archives( $args = '' ) {
 
 	$last_changed = wp_cache_get_last_changed( 'posts' );
 
-	$limit = $parsed_args['limit'];
+	/**
+	 * Filters the SQL LIMIT clause for retrieving archives.
+	 *
+	 * @since 7.0.0
+	 *
+	 * @param string $limit The limit of the query for the `wp_get_archives` function.
+	 * @param array  $parsed_args An array of default arguments.
```

Comment:
Nit: The descriptions should be aligned.

```suggestion
	 * @param string $limit       The limit of the query for the `wp_get_archives` function.
	 * @param array  $parsed_args An array of default arguments.
```